src/plugins/kibana/public/management/sections/indices/_create.js

```diff
@@ -3,21 +3,25 @@ import moment from 'moment';
 import { IndexPatternMissingIndices } from 'ui/errors';
 import 'ui/directives/validate_index_name';
 import 'ui/directives/auto_select_if_only_one';
-import PluginsKibanaSettingsSectionsIndicesRefreshKibanaIndexProvider from 'plugins/kibana/settings/sections/indices/_refresh_kibana_index';
+import RefreshKibanaIndex from 'plugins/kibana/management/sections/indices/_refresh_kibana_index';
 import uiRoutes from 'ui/routes';
 import uiModules from 'ui/modules';
-import createTemplate from 'plugins/kibana/settings/sections/indices/_create.html';
-
+import createTemplate from 'plugins/kibana/management/sections/indices/_create.html';
 
 uiRoutes
-.when('/settings/indices/', {
-  template: createTemplate
+.when('/management/data/index/', {
```

Comment:
That's correct. The thought for `/management/data/index` is that you're adding an existing index. I now see it could be confusing with how the word _index_ is generally used.